src/internal.c

```diff
@@ -8147,9 +8156,16 @@ static int BuildUserAuthRequestRsa(WOLFSSH* ssh,
             else {
                 int sigSz;
                 WLOG(WS_LOG_INFO, "Signing hash with RSA.");
+#ifdef WOLFSSH_TPM
+                wolfTPM2_SignHashScheme(ssh->ctx->tpmDev, ssh->ctx->tpmKey,
```

Comment:
I don't see any return code checking on this call? What if there is a TPM failure? What if `ssh->ctx->tpmDev` or `ssh->ctx->tpmKey` are not set?